[2022-08-01 13:57:32,835] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: weather_dag.weather manual__2022-08-01T13:57:30.245937+00:00 [queued]>
[2022-08-01 13:57:32,871] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: weather_dag.weather manual__2022-08-01T13:57:30.245937+00:00 [queued]>
[2022-08-01 13:57:32,873] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-01 13:57:32,876] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-01 13:57:32,878] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-01 13:57:32,923] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): weather> on 2022-08-01 13:57:30.245937+00:00
[2022-08-01 13:57:32,937] {standard_task_runner.py:52} INFO - Started process 610 to run task
[2022-08-01 13:57:32,949] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'weather', 'manual__2022-08-01T13:57:30.245937+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp9m6b_m_6', '--error-file', '/tmp/tmpusbk_ng_']
[2022-08-01 13:57:32,953] {standard_task_runner.py:80} INFO - Job 14: Subtask weather
[2022-08-01 13:57:33,321] {task_command.py:371} INFO - Running <TaskInstance: weather_dag.weather manual__2022-08-01T13:57:30.245937+00:00 [running]> on host 43173a86041a
[2022-08-01 13:57:33,466] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=weather
AIRFLOW_CTX_EXECUTION_DATE=2022-08-01T13:57:30.245937+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-01T13:57:30.245937+00:00
[2022-08-01 13:57:33,472] {logging_mixin.py:115} INFO - ETL Pipeline
[2022-08-01 13:57:33,473] {logging_mixin.py:115} INFO - Extraction stage started...
[2022-08-01 13:57:34,050] {logging_mixin.py:115} INFO - Transform stage started...
[2022-08-01 13:57:34,095] {logging_mixin.py:115} INFO -              Daytime         Day  Hours  Temperature  is Day  Humidity
0   2022-07-28 00:00  2022-07-28  00:00         74.8       0        88
1   2022-07-28 01:00  2022-07-28  01:00         74.7       0        88
2   2022-07-28 02:00  2022-07-28  02:00         74.5       0        88
3   2022-07-28 03:00  2022-07-28  03:00         74.3       0        89
4   2022-07-28 04:00  2022-07-28  04:00         74.3       0        88
5   2022-07-28 05:00  2022-07-28  05:00         74.3       0        88
6   2022-07-28 06:00  2022-07-28  06:00         74.3       0        88
7   2022-07-28 07:00  2022-07-28  07:00         76.5       1        80
8   2022-07-28 08:00  2022-07-28  08:00         78.7       1        73
9   2022-07-28 09:00  2022-07-28  09:00         81.0       1        65
10  2022-07-28 10:00  2022-07-28  10:00         84.0       1        63
11  2022-07-28 11:00  2022-07-28  11:00         87.1       1        60
12  2022-07-28 12:00  2022-07-28  12:00         90.1       1        58
13  2022-07-28 13:00  2022-07-28  13:00         88.6       1        59
14  2022-07-28 14:00  2022-07-28  14:00         87.1       1        60
15  2022-07-28 15:00  2022-07-28  15:00         85.6       1        61
16  2022-07-28 16:00  2022-07-28  16:00         84.5       1        64
17  2022-07-28 17:00  2022-07-28  17:00         83.4       1        66
18  2022-07-28 18:00  2022-07-28  18:00         82.2       1        69
19  2022-07-28 19:00  2022-07-28  19:00         80.6       1        74
20  2022-07-28 20:00  2022-07-28  20:00         79.0       0        78
21  2022-07-28 21:00  2022-07-28  21:00         77.4       0        83
22  2022-07-28 22:00  2022-07-28  22:00         76.6       0        84
23  2022-07-28 23:00  2022-07-28  23:00         75.9       0        84
[2022-08-01 13:57:34,102] {logging_mixin.py:115} INFO - Data is valid, proceed to load stage.
[2022-08-01 13:57:34,105] {logging_mixin.py:115} INFO - Load stage started...
[2022-08-01 13:57:34,110] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/script/WeatherETL.py", line 150, in run_weather_etl
    conn = sqlite3.connect('../weather_data.sqlite')
sqlite3.OperationalError: unable to open database file
[2022-08-01 13:57:34,144] {taskinstance.py:1415} INFO - Marking task as UP_FOR_RETRY. dag_id=weather_dag, task_id=weather, execution_date=20220801T135730, start_date=20220801T135732, end_date=20220801T135734
[2022-08-01 13:57:34,179] {standard_task_runner.py:92} ERROR - Failed to execute job 14 for task weather (unable to open database file; 610)
[2022-08-01 13:57:34,248] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-08-01 13:57:34,513] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
